common/app/model/Formats.scala

```diff
@@ -110,7 +109,7 @@ object MetaDataFormat {
         part2.contentType,
         part2.hasHeader,
         part2.schemaType,
-        part2.cacheSeconds,
```

Comment:
The only reason to keep the cache seconds in the pressed json is to keep serialisation simple. So we wouldn't need too many rules describing which fields get 'slimmed', which ones become default, etc. Here, the deserialisation is a bit more complicated.